src/app/login/login.component.ts

```diff
@@ -54,12 +54,12 @@ export class LoginComponent implements OnInit {
               this.helper.localStorageSet('firstName', this.user.firstName);
               this.helper.localStorageSet('lastName', this.user.lastName);
               this.helper.localStorageSet('address', this.user.address);
-              this.helper.localStorageSet('creditCardNumber', this.user.creditCardNumber);
-              this.helper.localStorageSet('cvv', this.user.cvv);
+              this.helper.localStorageSet('creditCardNumber', this.user.creditCardNumber + '');
+              this.helper.localStorageSet('cvv', this.user.cvv + '');
               this.helper.localStorageSet('role', this.user.role);
-              this.helper.localStorageSet('userId', (this.user.userId + ""));
+              this.helper.localStorageSet('userId', (u.userId + ""));
               console.log('User is logged in');
-              this.router.navigate(['/products']);
+              this.router.navigate(['']);
```

Comment:
shouldnt this be this.router.navigate(['/'])

Reply:
'' acts as home